2024-03-10 09:23:55,162 INFO    MainThread:5900 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-03-10 09:23:55,162 INFO    MainThread:5900 [wandb_setup.py:_flush():76] Configure stats pid to 5900
2024-03-10 09:23:55,162 INFO    MainThread:5900 [wandb_setup.py:_flush():76] Loading settings from /Users/yunbo/.config/wandb/settings
2024-03-10 09:23:55,162 INFO    MainThread:5900 [wandb_setup.py:_flush():76] Loading settings from /Users/yunbo/Documents/GitHub/PFL_Optimiozation/wandb/settings
2024-03-10 09:23:55,162 INFO    MainThread:5900 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-10 09:23:55,162 INFO    MainThread:5900 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-10 09:23:55,162 INFO    MainThread:5900 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'Local_learning/MLP_locl.py', 'program_abspath': '/Users/yunbo/Documents/GitHub/PFL_Optimiozation/Local_learning/MLP_locl.py', 'program': '/Users/yunbo/Documents/GitHub/PFL_Optimiozation/Local_learning/MLP_locl.py'}
2024-03-10 09:23:55,162 INFO    MainThread:5900 [wandb_init.py:_log_setup():526] Logging user logs to /Users/yunbo/Documents/GitHub/PFL_Optimiozation/wandb/run-20240310_092355-apj0js16/logs/debug.log
2024-03-10 09:23:55,162 INFO    MainThread:5900 [wandb_init.py:_log_setup():527] Logging internal logs to /Users/yunbo/Documents/GitHub/PFL_Optimiozation/wandb/run-20240310_092355-apj0js16/logs/debug-internal.log
2024-03-10 09:23:55,162 INFO    MainThread:5900 [wandb_init.py:init():566] calling init triggers
2024-03-10 09:23:55,162 INFO    MainThread:5900 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'region': 'US Center'}
2024-03-10 09:23:55,162 INFO    MainThread:5900 [wandb_init.py:init():616] starting backend
2024-03-10 09:23:55,162 INFO    MainThread:5900 [wandb_init.py:init():620] setting up manager
2024-03-10 09:23:55,164 INFO    MainThread:5900 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-03-10 09:23:55,164 INFO    MainThread:5900 [wandb_init.py:init():628] backend started and connected
2024-03-10 09:23:55,165 INFO    MainThread:5900 [wandb_init.py:init():720] updated telemetry
2024-03-10 09:23:55,186 INFO    MainThread:5900 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-10 09:23:56,516 INFO    MainThread:5900 [wandb_run.py:_on_init():2254] communicating current version
2024-03-10 09:23:56,568 INFO    MainThread:5900 [wandb_run.py:_on_init():2263] got version response upgrade_message: "wandb version 0.16.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-03-10 09:23:56,568 INFO    MainThread:5900 [wandb_init.py:init():804] starting run threads in backend
2024-03-10 09:24:02,095 INFO    MainThread:5900 [wandb_run.py:_console_start():2233] atexit reg
2024-03-10 09:24:02,095 INFO    MainThread:5900 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-03-10 09:24:02,095 INFO    MainThread:5900 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-03-10 09:24:02,095 INFO    MainThread:5900 [wandb_run.py:_redirect():2178] Redirects installed.
2024-03-10 09:24:02,096 INFO    MainThread:5900 [wandb_init.py:init():847] run started, returning control to user process
2024-03-10 09:24:04,780 INFO    MainThread:5900 [wandb_run.py:_finish():1962] finishing run digitalai/CFL_local_MLP3/apj0js16
2024-03-10 09:24:04,780 INFO    MainThread:5900 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-03-10 09:24:04,780 INFO    MainThread:5900 [wandb_run.py:_restore():2185] restore
2024-03-10 09:24:04,781 INFO    MainThread:5900 [wandb_run.py:_restore():2191] restore done
2024-03-10 09:24:09,455 INFO    MainThread:5900 [wandb_run.py:_footer_history_summary_info():3857] rendering history
2024-03-10 09:24:09,456 INFO    MainThread:5900 [wandb_run.py:_footer_history_summary_info():3889] rendering summary
2024-03-10 09:24:09,463 INFO    MainThread:5900 [wandb_run.py:_footer_sync_info():3816] logging synced files
